.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
What do you think of changing this to "Report a defect to help us fix it"?

Reply:
Sure!